src/App.js

```diff
@@ -1,25 +1,73 @@
 import React from 'react';
+import { Clock } from './Clock';
 
 import './App.scss';
 
-const App = () => {
-  setInterval(() => {
-    const date = new Date();
-
-    // eslint-disable-next-line
-    console.log(date.toLocaleTimeString());
-  }, 1000);
-
-  return (
-    <div className="App">
-      <h1>React clock</h1>
-      <p>
-        Current time:
-        {' '}
-        {/* Print the time here instead of DevTools */}
-      </p>
-    </div>
-  );
-};
+function letRandomName() {
+  return Math.random();
+}
+
+export class App extends React.Component {
+  state = {
+    isClockVisible: true,
+    clockName: null,
+  }
+
+  render() {
+    return (
+      <div className="App">
+        <Clock
+          isClockVisible={this.state.isClockVisible}
+          name={this.state.clockName}
+        />
```

Comment:
use conditional rendering for Clock

Reply:
conditional rendering looks like this:
```suggestion
        {isClockVisible && (
          <Clock
            isClockVisible={this.state.isClockVisible}
            name={this.state.clockName}
          />)}
```